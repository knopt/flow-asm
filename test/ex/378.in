1
45
2.6159625674071414
3.062551997744997 
-0.047576905630188904 
-2.3630082160408055 
-1.7328503487580136 
7.249540487256407 
6.794782153844633 
6.278381687093226 
-8.186590064958938 
5.196605141067781 
-1.0515764429377565 
-1.5043738289134208 
-8.967746872953962 
2.9818569671807715 
-1.1563908463874295 
7.205712294773335 
3.1983890047731904 
8.078739358469125 
-9.26770454094491 
-8.415436262961649 
-1.7869991082795451 
2.4793507121845337 
9.722187043298788 
2.864210375728394 
2.683761081512152 
-1.6441680889854169 
-4.936505356779488 
-6.862076124105208 
2.1803780578446474 
-0.9260948187521745 
-0.6270155204801462 
4.244793567477849 
0.6908944275735394 
4.383619248119192 
-9.276791828024084 
-2.2775230008524545 
2.5402526940747485 
5.215032495717233 
-3.048651901862356 
9.685410227653776 
-0.6847202734813997 
-3.387441802413216 
3.078482562150315 
7.100948126445509 
1.9749174479356935 
-1.9249141173099034 

5.660012378784849 7.169666725937567 9.571374065516189 -5.116625037462561 -6.651066497446106 -6.830184298052044 -9.13772174088287 8.619657525383772 5.173913480336044 6.47494936835497 -5.108036802933269 6.531237660892749 -9.784033748487708 7.438298142232167 -6.4250911599081295 6.5928094526566206 -6.174368294438775 -8.035745122295054 8.027329166896628 -8.797656063627823 9.974868103006035 -7.918637686369461 7.793150703517796 5.704697145962317 0.7525064810308066 6.1840874992876 -1.9713363231243441 -2.7160537482698555 6.302748818670683 -6.059755560044165 -7.31287137059854 1.871375298512742 5.427461969209077 4.110375380303148 3.552581266448982 3.125182237544733 -7.054115605706568 -7.260355792632691 6.622606472858099 6.745664357455819 2.6199706290160094 3.122289580208591 -1.9793513641383758 9.063688893425628 -1.974006204844894 
