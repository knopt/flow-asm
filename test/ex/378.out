12.465488722826152 
61.00501702641986 
54.296202411098385 
27.97000910744277 
-122.98261513200691 
-105.86154413236076 
-98.70533614350977 
141.09013915850267 
-33.91902467289981 
39.47214161295215 
12.623644888883412 
90.3221185049104 
-51.55552267068981 
17.675130327464203 
-61.80481085234457 
-14.365558729404398 
-133.3933436964968 
94.8974176249176 
49.701849290765516 
30.13677241597883 
-26.827255100696817 
-77.69803174461889 
12.449375585400476 
8.050646889690931 
47.03729247957686 
50.36897249700304 
79.5979738537635 
-42.486404492016675 
8.781156663392338 
-2.2387146645659053 
-81.1960507451646 
14.189016878705997 
-45.56761306246253 
151.81542304792384 
38.11088486736681 
-23.985885233475194 
-93.59733138472816 
55.684514063292255 
-110.7862275573904 
66.57118357679462 
79.84959872297128 
-17.629470952044976 
-45.858464610517174 
3.052113527655038 
36.894250357399535 
